scim/resource_user.go

```diff
@@ -23,82 +23,47 @@ const (
 	userAttributes = "userName,displayName,active,externalId,entitlements"
 )
 
-func setCommonUserFields(d *schema.ResourceData, user User, username string) {
-	d.Set("display_name", user.DisplayName)
-	d.Set("active", user.Active)
-	d.Set("external_id", user.ExternalID)
-	d.Set("home", fmt.Sprintf("/Users/%s", username))
```

Comment:
We need to make sure that all these fields are set in both user and SP. I recently fixed the problem in exporter that was caused by missing `external_id` when you import the resource